open-agent-builder/app/page.tsx

```
@@ -0,0 +1,263 @@
+          <HeaderWrapper>
+            <div className="max-w-[900px] mx-auto w-full flex justify-between items-center">
+              <div className="flex gap-24 items-center">
+                <HeaderBrandKit />
+              </div>
+              
+              <div className="flex gap-8 items-center">
+                {/* GitHub Template Button */}
+                <a
+                  className="contents"
+                  href="https://github.com/firecrawl/firecrawl"
+                  target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)